src/containers/allUserView.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toast } from 'react-toastify';
+import {
+  getAllUserAction, followAction, getFollowingAction, unfollowAction,
+} from '../redux/actions/followerAction';
+import { getDataThunkPrivate, getDataThunk, postDataThunkPrivate } from '../redux/thunks/index';
+import ViewAllUserCard from '../components/viewAllUserCard';
+import ProfileView from '../components/profilePopup';
+import Layout from '../components/Layout/Layout';
+
+export class AllUserView extends Component {
+    state = {
+      image: '',
+      username: '',
+      firstName: '',
+      lastName: '',
+      dob: '',
+      bio: '',
+      message: '',
+    };
+
+    componentDidMount = async () => {
+      await this.props.getDataThunkPrivate('get', '/allusers/', getAllUserAction);
+      await this.props.getDataThunkPrivate('get', 'profiles/following', getFollowingAction);
+    }
+
+    handelOnlick = (use) => () => {
+      this.setState({
+        image: use.image,
+        username: use.username,
+        firstName: use.firstName,
+        lastName: use.lastName,
+        dob: use.dob,
+        bio: use.bio,
+      });
+      const modal = document.getElementById('myModal');
+      modal.classList.remove('modal');
+      modal.classList.add('modalsd');
+    };
+
+    removeModel = async () => {
+      const modal = document.getElementById('myModal');
+      modal.classList.remove('modalsd');
+      modal.classList.add('modal');
+    };
+
+    followerEvent = (user) => async () => {
+      await this.props.postDataThunkPrivate('post', `profiles/${user.username}/follow`, followAction, null);
+      // eslint-disable-next-line no-unused-expressions
+      this.props.followMessage
+        ? (toast.success(`${this.props.followMessage}!`), window.location.reload())
+        : toast.error(`you already follow ${user.username}!`);
+    }
+
+    onKeyUpHandle = () => {
+      const { value } = document.getElementById('search-btn');
+      const username = document.getElementsByClassName('usernamevalue');
+      const userCard = document.getElementsByClassName('cards');
+      // eslint-disable-next-line no-plusplus
```

Comment:
Why did you disable linting here?

Reply:
I wanted to use for loop but ESLint was not allowing it.